doc/api/next_api_changes/behaviour.rst

```diff
@@ -147,3 +147,16 @@ The parameter ``s`` to `.Axes.annotate` and  `.pyplot.annotate` is renamed to
 The old parameter name remains supported, but
 support for it will be dropped in a future Matplotlib release.
 
+`Axes.vlines` and `Axes.hlines` `color` parameter will now default to configured rcparams `lines.color`
```

Comment:
These either needs to be qualified more `.Axes.vlines` should work or use double ticks.